bokeh/models/contour_renderer.py

```diff
@@ -0,0 +1,120 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2022, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' Models (mostly base classes) for the various kinds of renderer
+types that Bokeh supports.
+
+'''
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Bokeh imports
+from ..core.properties import Float, Instance, Seq
+from ..core.types import Unknown
+from .annotations import ContourColorBar
+from .glyphs import MultiLine, MultiPolygons
+from .renderers import DataRenderer, GlyphRenderer
+from .sources import ColumnDataSource
+from .tickers import FixedTicker
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+__all__ = (
+    'ContourRenderer',
+)
+
+#-----------------------------------------------------------------------------
+# General API
+#-----------------------------------------------------------------------------
+
+#-----------------------------------------------------------------------------
+# Dev API
+#-----------------------------------------------------------------------------
+
+_DEFAULT_CONTOUR_LINE_RENDERER = lambda: GlyphRenderer(
```

Comment:
I tried to use `InstanceDefault` here but in the end kept with this. This comment about `GraphRenderer` is relevant:
https://github.com/bokeh/bokeh/blob/2f617f55427e35bbfde531340b7dfadf2d3c3fd4/bokeh/models/renderers.py#L275-L280